packages/components/src/components/domainproperties/list/ListPropertiesAdvancedSettings.tsx

```diff
@@ -591,7 +615,7 @@ export class AdvancedSettings extends React.PureComponent<AdvancedSettingsProps,
 
                     <Modal.Body>
                         <SettingsContainer
-                            title="Field used for display title"
+                            title="Default Display Field"
                             tipTitle={'Display Title Field'}
```

Comment:
I think we should update the tipTitle so that it matches the title as well. I think you can do that by just removing the tipTitle prop so that it will use the title prop.